apps/callscreen/js/call_screen.js

```diff
@@ -461,6 +461,10 @@ var CallScreen = {
     this.mergeButton.classList.add('hide');
   },
 
+  showOnHoldAndMergeContainer: function cs_showOnHoldAndMergeContainer() {
+    this.holdAndMergeContainer.style.display = 'visible';
```

Comment:
Aw, right. In fact this is something I wanted to check and then just forgot to do.